packages/client/src/modules/post/components/PostCommentsView.jsx

```diff
@@ -13,57 +13,55 @@ export default class PostCommentsView extends React.PureComponent {
     addComment: PropTypes.func.isRequired,
     editComment: PropTypes.func.isRequired,
     deleteComment: PropTypes.func.isRequired,
-    onCommentSelect: PropTypes.func.isRequired,
-    onFormSubmitted: PropTypes.func.isRequired,
-    subscribeToMore: PropTypes.func.isRequired
+    subscribeToMore: PropTypes.func.isRequired,
+    addCommentClient: PropTypes.func.isRequired
   };
 
   keyExtractor = item => item.id;
 
   renderItem = ({ item: { id, content } }) => {
-    const { comment, deleteComment, onCommentSelect } = this.props;
+    const { comment, deleteComment, addCommentClient } = this.props;
```

Comment:
Why to call the parameter `addCommentClient`, what does it means? Please use better name here